### DISCUSSION TITLE
Anyway to connect to code server from a raspberry pi 4 to a iPad Pro with out any connection?

### DISCUSSION THREAD

**Discussion by ZeroAmp** [coder/code-server#5130] [Q&A]
Hello sorry if this is a dumb question but I haven’t been able to figure this out. I’m running code server on a raspberry pi 4 and connecting to it on an iPad Pro. Is there a way I can do this though the usb c without internet connection. I need to be able to have access offline while on the go. 
Thanks

**Comment by jsjoeio** [comment]
Not a dumb question! I'm trying to think how you would do that 🤔 I am guessing you could SSH into the Raspberry Pi on iPad using iSH but I'm guessing that would require a connection. I can't think of a way to do this otherwise.

Unless...you somehow figure out how to install code-server _directly_ on the iPad inside iSH or another terminal app like Blink. I haven't tried that though.

**Reply by lg-nineoneninethreenine** [reply]
Well, why not expose the port on your router? Then, if configured properly, you can access it via it's router IP.

**Reply by jsjoeio** [reply]
Oh that's a good point! Maybe that would work
